spec/factories/teams.rb

```diff
@@ -36,6 +36,12 @@
       end
     end
 
+    trait :without_terms do
```

Comment:
Is `without_terms` the right name for this? I feel it should be `:with_conference_preference`